frame/evm/src/lib.rs

```diff
@@ -935,32 +935,13 @@ impl<T: Config> Pallet<T> {
 	/// Remove an account.
 	pub fn remove_account(address: &H160) {
 		if <AccountCodes<T>>::contains_key(address) {
-			// Remember to call `dec_sufficients` when clearing Suicided.
-			<Suicided<T>>::insert(address, ());
```

Comment:
Since the behavior of the self-destruct has changed in this upgrade, there are still some places using Suicided storage. I think we can clean all those areas, including the Suicided storage item.

Reply:
The suicided storage can be removed, at moonbeam we already performed the [migration](https://github.com/moonbeam-foundation/moonbeam/pull/2561) to remove all storage of suicided contracts pre-cancun.

Reply:
@TarekkMA  Would you like to update the PR description with a note reminding users to clean the Suicided storage and link it to the Moonbeam one? Then we can remove the relevant storage items and related content in the PR.

Reply:
Done

Reply:
Please also remove the storage and associated staff.